envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -31,6 +31,7 @@ class EnvoyContainer(
         private const val EXTRA_DIR = "envoy/extra"
         private const val EXTRA_DIR_DEST = "/etc/envoy/extra"
 
+        const val ENVOY_UID_ENV_NAME = "ENVOY_UID"
```

Comment:
what does this do?

Reply:
https://www.envoyproxy.io/docs/envoy/latest/start/docker#permissions-for-running-docker-envoy-containers-as-a-non-root-user